src/streamer.rs

```diff
@@ -394,6 +397,18 @@ fn broadcast(
             win[pos] = Some(b);
         }
     }
+
+    // Fill in the coding blob data from the window data blobs
+    #[cfg(feature = "erasure")]
```

Comment:
Can you add this to the list of `--features` anywhere that we currently mention `cuda`?